tests/src/test/scala/com/nvidia/spark/rapids/CastOpSuite.scala

```diff
@@ -56,6 +56,7 @@ class CastOpSuite extends GpuExpressionTestSuite {
     case (TimestampType | DateType, _: NumericType) => true
     case (TimestampType | DateType, BooleanType) => true
     case (StringType, TimestampType) => true
+    case (FloatType, IntegerType) => true
```

Comment:
The disable I understand, but why was this change made?

Reply:
I updated issue https://github.com/NVIDIA/spark-rapids/issues/1271 explaining the Spark change and why it broke our tests

Reply:
Sorry, you are asking about this code change. This is a method that determines which CASTs to skip against 3.1.0 so this is just adding another one to that list.

Reply:
```scala
  def should310SkipAnsiCast(from: DataType, to: DataType): Boolean = (from, to) match {
    case (_: NumericType, TimestampType | DateType) => true
    case (BooleanType, TimestampType | DateType) => true
    case (TimestampType | DateType, _: NumericType) => true
    case (TimestampType | DateType, BooleanType) => true
    case (StringType, TimestampType) => true
    case (FloatType, IntegerType) => true
    case _ => false
  }
```